tfbpmodeling/interface.py

```diff
@@ -22,6 +23,77 @@
 logger = logging.getLogger("main")
 
 
+def generate_stage_result_table(tf_dir):
```

Comment:
Move this to its own file (name the file the same as the function) and write a test for it